recipe/meta.yaml

```diff
@@ -14,10 +14,13 @@ source:
     sha256: a9598abd5c697e4a49a512b285c5747413d1f45155a12ecb1f5ee389c0459afa
     folder: ucx
     patches:
+      - patches/PR_4137.patch
+      - patches/PR_4366.patch
       - patches/PR_4383.patch
       - patches/PR_4384.patch
+      - patches/PR_4431.patch
```

Comment:
These have not been backported to `1.7.x` yet, but the plan is to backport all of them. Linking the PRs below for reference/tracking purposes.

xref: https://github.com/openucx/ucx/pull/4137
xref: https://github.com/openucx/ucx/pull/4366
xref: https://github.com/openucx/ucx/pull/4431